codemp/rd-rend2/tr_ghoul2.cpp

```diff
@@ -3953,9 +3953,16 @@ qboolean R_LoadMDXM(model_t *mod, void *buffer, const char *mod_name, qboolean &
 
 		CModelCache->StoreShaderRequest(mod_name, &surfInfo->shader[0], &surfInfo->shaderIndex);		
 
+	// [Monsterovich] gcc throws an error here and Idk why but my custom code works!
```

Comment:
Not too sure about this. I understand the reason for the change but I think it can be done in a cleaner way. Maybe we can just change the `offsetof` below.

```c++
surfInfo = (mdxmSurfHierarchy_t *)((byte *)surfInfo + offsetof(mdxmSurfHierarchy_t, childIndexes) + sizeof(int) * surfInfo->numChildren);
```

Assuming `childIndexes` is an array of `int`s.